### ISSUE TITLE
Email Reminder

### ISSUE THREAD

**Issue by allisonanne** [SIU-CS-435/RARE-production#5]
As a user I want to receive email reminders so that I can stay updated on my tasks on the go.

1. User received email notification.
2. User was provided the name of the task and the time remaining.

**Comment by ethanx94** [comment]
Requires: Deadline completed first, Lists and Tasks (for userID)
Add checkbox option in add new task: Enable Reminders
Use SendGrid to email to the user that a new task is created
Use SendGrid to email to the user if the deadline is tomorrow

**Comment by allisonanne** [comment]
Decided too complex to handle for now
